app/modules/users/helpers.js

```diff
@@ -30,6 +30,11 @@ export default function (app) {
       return userModel.toJSON()
     },
 
+    async getUsersData (ids) {
```

Comment:
```suggestion
    async getUsersModelsFromIds (ids) {
```

Reply:
The "Json" in that name would be propagating the Backbone lie (that `toJSON` method does not return json but an object). But I agree the "Data" part in the name is smelly. The dirty part in this, which is actually the point of reusing that legacy Backbone model based `getUsersModels` function, is that `getUsersModels` builds a local in-memory cache of users. That could indeed be worth making explicit.

Reply:
or `getSerializedUsers` ? So data naming can apply to other frameworks than bb. 

Its only to signify that this user object is not the user doc coming straight from the server..

Reply:
renamed and refactored in 3b671b342